src/smartpeak/include/SmartPeak/core/sources.cmake

```diff
@@ -9,6 +9,7 @@ set(sources_list_h
 	CastValue.h
 	ConsoleHandler.h
 	EventDispatcher.h
+  FeaturesObservable.h
```

Comment:
super minor: it loolk we have spaces here where tabs are used in the other lines.

Reply:
i couldn't spot the space with the editor I am using so I changed its location